### ISSUE TITLE
ImportWarning: _SixMetaPathImporter.find_spec() not found; falling back to find_module()

### ISSUE THREAD

**Issue by dinishi** [mwilliamson/python-mammoth#144]
Hey,
I am running python 3.11.9 and I'm getting a warning

``` 
.venv/lib/python3.11/site-packages/mammoth/docx/__init__.py:14: in <module>
    from .files import Files
        Numbering  = <class 'mammoth.docx.numbering_xml.Numbering'>
        Relationships = <class 'mammoth.docx.relationships_xml.Relationships'>
        Styles     = <class 'mammoth.docx.styles_xml.Styles'>
        __builtins__ = <builtins>
        __cached__ = '/.../.venv/lib/python3.11/site-packages/mammoth/docx/__pycache__/__init__.cpython-311.pyc'
        __doc__    = None
        __file__   = '/.../.venv/lib/python3.11/site-packages/mammoth/docx/__init__.py'
        __loader__ = <_frozen_importlib_external.SourceFileLoader object at 0x71408ccd2f50>
        __name__   = 'mammoth.docx'
        __package__ = 'mammoth.docx'
        __path__   = ['/.../.venv/lib/python3.11/site-packages/mammoth/docx']
        __spec__   = ModuleSpec(name='mammoth.docx', loader=<_frozen_importlib_external.SourceFileLoader object at 0x71408ccd2f50>, origin=...py', submodule_search_locations=['/.../.venv/lib/python3.11/site-packages/mammoth/docx'])
        cobble     = <module 'cobble' from '/.../.venv/lib/python3.11/site-packages/cobble/__init__.py'>
        comments_xml = <module 'mammoth.docx.comments_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/comments_xml.py'>
        content_types_xml = <module 'mammoth.docx.content_types_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/content_types_xml.py'>
        document_xml = <module 'mammoth.docx.document_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/document_xml.py'>
        empty_content_types = <mammoth.docx.content_types_xml._ContentTypes object at 0x71408cb463d0>
        lists      = <module 'mammoth.lists' from '/.../.venv/lib/python3.11/site-packages/mammoth/lists.py'>
        notes_xml  = <module 'mammoth.docx.notes_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/notes_xml.py'>
        numbering_xml = <module 'mammoth.docx.numbering_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/numbering_xml.py'>
        os         = <module 'os' (frozen)>
        partial    = <class 'functools.partial'>
        read_comments_xml_element = <function read_comments_xml_element at 0x71408cbd3920>
        read_content_types_xml_element = <function read_content_types_xml_element at 0x71408cbd0a40>
        read_document_xml_element = <function read_document_xml_element at 0x71408cbb7560>
        read_endnotes_xml_element = functools.partial(<function _read_notes at 0x71408cbd3740>, 'endnote')
        read_footnotes_xml_element = functools.partial(<function _read_notes at 0x71408cbd3740>, 'footnote')
        read_numbering_xml_element = <function read_numbering_xml_element at 0x71408cbd1800>
        read_relationships_xml_element = <function read_relationships_xml_element at 0x71408cbd14e0>
        read_styles_xml_element = <function read_styles_xml_element at 0x71408cbd1b20>
        relationships_xml = <module 'mammoth.docx.relationships_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/relationships_xml.py'>
        results    = <module 'mammoth.results' from '/.../.venv/lib/python3.11/site-packages/mammoth/results.py'>
        styles_xml = <module 'mammoth.docx.styles_xml' from '/.../.venv/lib/python3.11/site-packages/mammoth/docx/styles_xml.py'>
        zips       = <module 'mammoth.zips' from '/.../.venv/lib/python3.11/site-packages/mammoth/zips.py'>
.venv/lib/python3.11/site-packages/mammoth/docx/files.py:4: in <module>
    from urllib2 import urlopen
E   ImportWarning: _SixMetaPathImporter.find_spec() not found; falling back to find_module()

```

I suspect it's related to `six.py` file in `cobble` package missing `find_spec()` which supersedes `find_module()/find_loader()` [python deprecation notes](https://docs.python.org/3.10/whatsnew/3.10.html#deprecated), but I might be wrong.

Would it be possible to have this fixed?

**Comment by mwilliamson** [comment]
I've just released a new version of cobble (0.1.4) that no longer uses six, which I think should fix this error?

**Comment by dinishi** [comment]
Yes, thank you :pray:
